React/tic-tac-toe/src/index.js

```diff
@@ -1,23 +1,42 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { AppContainer } from 'react-hot-loader';
+import { Provider } from 'react-redux';
+import { BrowserRouter, Switch, Route, Redirect } from 'react-router-dom';
 
+import AuthorizedRoute from './app/components/AuthorizedRoute';
+import store from './redux/store';
 import App from './app';
 import './config/i18n';
 import './scss/application.scss';
 import { register } from './serviceWorker';
+import Login from './app/screens/Login';
+import { saveState } from './utils/localStorage';
 
+// Para persistir el estado en el storage del browser
+store.subscribe(() => {
+  saveState(store.getState());
+});
+// El provider lo agregué yo, y los routes
 const render = () => {
```

Comment:
delete comment

Reply:
> this function is not needed.. always we render only one component inside the render function of `ReactDOM`. Move the Provider and the rest to another file named app.js and re call it from here.

I think this function was created to avoid rewriting, as it's called again in line 45, for "hot module"